module_1_task_7/task_3.py

```diff
@@ -0,0 +1,18 @@
+a = True
+b = False
+c = True
+
+# becouse both a and b need be a true
+print( a and b )
+
+#becouse at least one value True
+print( a or b )
+
+#becouse b it's true
+print( not b )
+
+#becouse a and c ture, and b false,why true?becouse  at least one value True
+print( ( a and c ) or b )
+
+#becouse b or c becouse  at least one value True,and a true (b or c)  also true
+print( a and ( b or c ) )
```

Comment:
Here we do not need this spaces, they are a bit redundant)

```suggestion
print(a and b)

#becouse at least one value True
print(a or b)

#becouse b it's true
print(not b)

#becouse a and c ture, and b false,why true?becouse  at least one value True
print((a and c) or b)

#becouse b or c becouse  at least one value True,and a true (b or c)  also true
print(a and (b or c))
```